index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent<br>Works</h1>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
```

Comment:
- [ ] Please add the right sizes, margins, and padding to all your elements since this list of languages is overflowing past the card's border. The 'See Projects' button also goes beyond the border when there should be some space between it and the border. Please remember to fix this issue in all the cards.

  ![port71](https://user-images.githubusercontent.com/87394385/213251177-7ee1c0a4-bd81-4962-956d-4b626efb3f19.JPG)